app/Http/Controllers/ItemController.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Models\Imobil;
+use App\Services\ModelLogger;
+use Illuminate\Http\Request;
+
+class ItemController extends Controller
+{
+
+    public function index($imobilId, Request $request, ModelLogger $logger)
+    {
+
+        $imobil = Imobil::findOrFail($imobilId);
+
+        $logger->logModel2($user = $request->user(), $imobil);
```

Comment:
Not fixed.

Reply:
change from logModel2 in logModelItem 7d4ba19